src/main/kotlin/app/model/Commit.kt

```diff
@@ -20,6 +22,13 @@ class Commit : ProtoWrapper<Commit, CommitProtos.Commit> {
     var numLinesDelete: Int = 0
     // TODO(anatoly): add Stats.
 
+    constructor(revCommit: RevCommit) : this() {
+        this.id = DigestUtils.sha256Hex(revCommit.id.name)
+        this.author = Author(revCommit.authorIdent.name,
+                revCommit.authorIdent.emailAddress)
+        this.dateTimestamp = revCommit.commitTime
```

Comment:
What did we decide on versioning of commits? Because this would be a good place to register the time processing the commit.